ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/cli/Cli.scala

```diff
@@ -159,7 +159,19 @@ object Cli {
       .optional()
       .validate(v => Either.cond(v.length > 0, (), "HMAC secret must be a non-empty string"))
       .text("Enables JWT-based authorization, where the JWT is signed by HMAC256 with the given secret")
-      .action( (secret, config) => config.copy(authService = Some(AuthServiceJWT(HMAC256Verifier(secret).getOrElse(sys.error("Failed to create HMAC256 verifier"))))))
+      .action( (secret, config) => config.copy(authService = Some(AuthServiceJWT(HMAC256Verifier(secret).valueOr(err => sys.error(s"Failed to create HMAC256 verifier: $err"))))))
+
+    opt[String]("auth-jwt-rs256-crt")
```

Comment:
There is a bit of inconsistency here:
* the CLI option uses `crt` for certificate
* The validation error uses `Public key file`
* And the long text says X509 certificate file

Should this file contain the certificate or the public key?
I don't know what would be more common to "encounter" in the wild.

Reply:
I have renamed the method name and some strings to be more consistent.

As to which one is more common. I have no actual experience with this, but here is what the internet has told me:

There are several common formats that include RSA key information:
- [PKCS#1](https://en.wikipedia.org/wiki/PKCS_1): contains public key information
- [X.509](https://en.wikipedia.org/wiki/X.509): contains public key information and also additional info like issuer or a signature of the public key
- [PKCS#8](https://en.wikipedia.org/wiki/PKCS_8): contains private key information

There are two ways to serialize the above:
- [DER](https://en.wikipedia.org/wiki/X.690#DER_encoding), a binary format
- [PEM](https://en.wikipedia.org/wiki/Privacy-Enhanced_Mail), a text format. Basically a base64 encoded version of a DER file, with a header of the form:
  - `----BEGIN CERTIFICATE----` for X.509 data (`Certificate` structure)
  - `----BEGIN PUBLIC KEY----` for X.509 data (`SubjectPublicKeyInfo` structure, which is [a subset of the above](https://en.wikipedia.org/wiki/X.509#Structure_of_a_certificate))
  - `----BEGIN RSA PUBLIC KEY----` for PKCS#1 data
  - `----BEGIN PRIVATE KEY----` for PKCS#8 data
  - [... a dozen other things](https://stackoverflow.com/a/5356351/2289701)

File extensions [tell you very little about the content of the file](https://support.ssl.com/Knowledgebase/Article/View/19/0/der-vs-crt-vs-cer-vs-pem-certificates-and-how-to-convert-them).

Currently, the `--auth-jwt-rs256-crt` sandbox option supports X.509 (the `Certificate` kind, not the `SubjectPublicKeyInfo` kind) in both PEM and DER encoding.

Reply:
Thanks 👍